--- xdm-1.1.12/config/Xresources.in.orig	2006-08-14 20:12:34.000000000 +0200
+++ xdm-1.1.12/config/Xresources.in	2006-10-15 02:18:19.860305500 +0200
@@ -78,7 +78,7 @@
 #endif /* XPM */
 XHASHendif
 #ifdef XPM
-XHASHif PLANES >= 8
+XHASHif PLANES > 8
 xlogin*logoFileName: BITMAPDIR/**//XDM_PIXMAP
 XHASHelse
 xlogin*logoFileName: BITMAPDIR/**//XDM_BWPIXMAP
--- xdm-1.1.0/xdm-xinitrc-0.2/GiveConsole.orig	2000-10-29 20:59:44.000000000 +0100
+++ xdm-1.1.0/xdm-xinitrc-0.2/GiveConsole	2006-10-15 02:18:00.359086750 +0200
@@ -8,4 +8,4 @@
 # causing serious grief.
 #
 chown $USER /dev/console
-/usr/X11R6/bin/sessreg  -a -x "/etc/X11/xdm/Xservers" -l $DISPLAY -h "" $USER
+/usr/bin/sessreg  -a -x "/etc/X11/xdm/Xservers" -l $DISPLAY -h "" $USER
--- xdm-1.1.0/xdm-xinitrc-0.2/TakeConsole.orig	2000-10-29 20:59:44.000000000 +0100
+++ xdm-1.1.0/xdm-xinitrc-0.2/TakeConsole	2006-10-15 02:18:07.591538750 +0200
@@ -5,4 +5,4 @@
 #
 chmod 622 /dev/console
 chown root /dev/console
-/usr/X11R6/bin/sessreg -d -x "/etc/X11/xdm/Xservers" -l $DISPLAY -h "" $USER
+/usr/bin/sessreg -d -x "/etc/X11/xdm/Xservers" -l $DISPLAY -h "" $USER

