commit 4fbd4bfcca12a91dacb25fb08a6a3b07822b5845
Author: James Cloos <cloos@jhcloos.com>
Date:   Wed Aug 20 10:32:50 2008 -0400

    xaw8 is gone, use xaw7

diff --git a/configure.ac b/configure.ac
index 329c8e8..74f38fb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -305,7 +305,7 @@ AC_SUBST(XDM_PIXMAPDIR)
 
 # Packages used by multiple programs
 
-XAW_CHECK_XPRINT_SUPPORT(XDM_PRINT)
+PKG_CHECK_MODULES(XDM_PRINT, xaw7)
 PKG_CHECK_MODULES(DMCP, xdmcp)
 PKG_CHECK_MODULES(XLIB, x11)
 PKG_CHECK_MODULES(AUTH, xau)
