diff -urNp -x '*.orig' xdm-1.1.12.org/config/Xresources.in xdm-1.1.12/config/Xresources.in
--- xdm-1.1.12.org/config/Xresources.in	2019-03-02 22:06:13.000000000 +0000
+++ xdm-1.1.12/config/Xresources.in	2022-04-06 18:24:57.852216768 +0000
@@ -58,7 +58,7 @@ xlogin*shdColor: black
 xlogin*hiColor: black
 XHASHendif
 #ifdef XPM
-XHASHif PLANES >= 8
+XHASHif PLANES > 8
 xlogin*logoFileName: BITMAPDIR/**//XDM_PIXMAP
 XHASHelse
 xlogin*logoFileName: BITMAPDIR/**//XDM_BWPIXMAP
diff -urNp -x '*.orig' xdm-1.1.12.org/xdm-xinitrc-0.2/GiveConsole xdm-1.1.12/xdm-xinitrc-0.2/GiveConsole
--- xdm-1.1.12.org/xdm-xinitrc-0.2/GiveConsole	2000-10-29 19:59:44.000000000 +0000
+++ xdm-1.1.12/xdm-xinitrc-0.2/GiveConsole	2022-04-06 18:24:57.852216768 +0000
@@ -8,4 +8,4 @@
 # causing serious grief.
 #
 chown $USER /dev/console
-/usr/X11R6/bin/sessreg  -a -x "/etc/X11/xdm/Xservers" -l $DISPLAY -h "" $USER
+/usr/bin/sessreg  -a -x "/etc/X11/xdm/Xservers" -l $DISPLAY -h "" $USER
diff -urNp -x '*.orig' xdm-1.1.12.org/xdm-xinitrc-0.2/TakeConsole xdm-1.1.12/xdm-xinitrc-0.2/TakeConsole
--- xdm-1.1.12.org/xdm-xinitrc-0.2/TakeConsole	2000-10-29 19:59:44.000000000 +0000
+++ xdm-1.1.12/xdm-xinitrc-0.2/TakeConsole	2022-04-06 18:24:57.852216768 +0000
@@ -5,4 +5,4 @@
 #
 chmod 622 /dev/console
 chown root /dev/console
-/usr/X11R6/bin/sessreg -d -x "/etc/X11/xdm/Xservers" -l $DISPLAY -h "" $USER
+/usr/bin/sessreg -d -x "/etc/X11/xdm/Xservers" -l $DISPLAY -h "" $USER
